src/test/java/rx/internal/util/UtilityFunctionsTest.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package rx.internal.util;
+
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.*;
+
+import org.junit.Test;
+
+public class UtilityFunctionsTest {
+    @Test
+    public void testNotInstantiable() {
+        try {
+            Constructor<?> c = UtilityFunctions.class.getDeclaredConstructor();
```

Comment:
Personally I don't see the benefit of recreating that mechanism / check instead of just adding a `testCompile` dependency. Also Guava was recently added as a test dependency.